config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+  get '/users/:user_id/posts(.:format)', to: 'posts#index'
+  get '/users/:user_id/posts/:id(.:format)', to: 'posts#show'
+  get '/users(.:format)', to: 'users#index'
+  get '/users/:id(.:format)', to: 'users#show'
```

Comment:
- [ ] Nice adding the routes but it would be better to use the `resources` keyword it would shortcut a lot

Reply:
Thank you, I have implemented it